src/list.scss

```diff
@@ -1,980 +1,8 @@
 @import "./new-settings";
 @import "./mixins";
```

Comment:
We can probably remove new-settings and mixins here since it's inside of listDefinitions